Test 3

KGalia opened the pull request.

KGalia committed:
Pinia is installed. Variable todos was replaced by store.  When initializing the application, the list was filled with data.

KGalia committed:
Removed unnecessary code

ViktoriaStarik commented on vue-project/src/components/main/Todo.vue:
```diff
@@ -1,26 +1,25 @@
 <template>
   <div class="my-style">
     <router-link :to="{ name: 'TodoPage', params: {id: todo.id} }">
```
Наверно, надо props.todo.id

ViktoriaStarik commented on vue-project/src/components/details/TodoDetails.vue:
```diff
@@ -2,47 +2,32 @@
-
-      <TodoItemsListDetails
-          :items="this.todo.items"
+      <h2> {{ findTodo(props.id).title }} </h2>
```
Правильно будет не искать каждый раз нужный туду, а сохранить findTodo(props.id) в computed переменную при загрузке страницы

KGalia committed:
Removed unnecessary code.
Creating a Todo List Creation page. Creating a Button component. Creating a component Creating a Todo.

KGalia committed:
Revert "Removed unnecessary code."

This reverts commit 965ddbbfc604e4d0ba2b67400b3400abd0af324d.
